pxtblocks/blocklyloader.ts

```diff
@@ -755,12 +792,12 @@ namespace pxt.blocks {
             }
 
             let deleteOption = {
-                text: deleteList.length == 1 ? Blockly.Msg.DELETE_BLOCK :
-                    Blockly.Msg.DELETE_X_BLOCKS.replace('%1', String(deleteList.length)),
+                text: deleteList.length == 1 ? lf("Delete Block") :
+                    lf("Delete %1 Blocks").replace('%1', String(deleteList.length)),
                 enabled: deleteList.length > 0,
                 callback: function () {
                     if (deleteList.length < 2 ||
-                        window.confirm(Blockly.Msg.DELETE_ALL_BLOCKS.replace('%1',
+                        window.confirm(lf("Delete all %1 blocks?").replace('%1',
```

Comment:
``lf`` supports arguments which will be substitued to {0}, {1}, ...

See example at https://github.com/Microsoft/pxt/blob/c3f64ddcd6533ff8338346d12a7e291b03c7a7f1/webapp/src/appcache.ts#L13